java/ql/src/semmle/code/java/dataflow/ExternalFlow.qll

```diff
@@ -322,7 +327,8 @@ private predicate summaryModelCsv(string row) {
       "java.io;StringReader;false;StringReader;;;Argument[0];Argument[-1];taint",
       "java.io;CharArrayReader;false;CharArrayReader;;;Argument[0];Argument[-1];taint",
       "java.io;BufferedReader;false;BufferedReader;;;Argument[0];Argument[-1];taint",
-      "java.io;InputStreamReader;false;InputStreamReader;;;Argument[0];Argument[-1];taint"
+      "java.io;InputStreamReader;false;InputStreamReader;;;Argument[0];Argument[-1];taint",
+      "java.net;URLClassLoader;false;URLClassLoader;;;Argument[0];Argument[-1];taint"
```

Comment:
There are constructor overloads that take the `URL[]` parameter in position 1 also

Reply:
```suggestion
      "java.net;URLClassLoader;false;URLClassLoader;;;ArrayElement of Argument[0];Argument[-1];taint"
```
This will mean you don't need the `ArrayCreationExpr` step either